build.py

```diff
@@ -755,6 +761,15 @@ def armnn_tflite_cmake_args():
     ]
 
 
+def fastertransformer_cmake_args():
+    print(
+        "Warning: Fastertransformer backend is not officially supported part of Triton."
```

Comment:
Nit: Wording. "not an officially supported part" or "not officially supported." I also believe FasterTransformer uses CamelCase for their capitalization (i.e. "FasterTransformer").